### PR TITLE
[libpcap] new port only linux

### PR DESCRIPTION
**Pull request by ayum**

### CODE REVIEW

File: ports/libpcap/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+)
+
+vcpkg_extract_source_archive_ex(
+    OUT_SOURCE_PATH SOURCE_PATH
+    ARCHIVE ${SOURCE_ARCHIVE_PATH}
+    REF 1.9.0
+)
+
+if(NOT VCPKG_CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    message(FATAL_ERROR "Package only supports linux platform.")
```
Comment (Voskrese) [inline ports/libpcap/portfile.cmake]: message(FATAL_ERROR "Current PORT ${PORT} supports only linux platform.")

**Comment by Voskrese** [comment]
Noticed with such tags "FATAL_ERROR" Check will not be passed, will be failed, fo Vcpkg-PR-Eager

_As something strange_

**Comment by grdowns** [comment]
Thanks for the new port, @Voskrese! If I understand you correctly, the Eager build fails only for regressions (and also if the port does not work for `arm-uwp`, unfortunately). As long as your new port does not cause other port builds to fail, the build will pass. Currently the CI build is failing due to `arm-uwp`, which is a sort-of false positive.

**Comment by Voskrese** [comment]
@grdowns It's not my port) just notes. This is the port @almikhayl

File: ports/libpcap/portfile.cmake (hunk 1 above)
Comment (grdowns) [reply ports/libpcap/portfile.cmake]: Successful? This port does not get past the extraction step on Windows for me

Comment (Voskrese) [reply ports/libpcap/portfile.cmake]: In VCPKG there is no port for WINDOWS, I collected it separately

**Comment by Voskrese** [comment]
How to escape/remove the assembly under -->> `VCPKG_CMAKE_SYSTEM_NAME "ARM-UWP"` ?

**Comment by grdowns** [comment]
> How to escape/remove the assembly under -->> `VCPKG_CMAKE_SYSTEM_NAME "ARM-UWP"` ?

**Comment by Voskrese** [comment]
stumbled across - port cmake
https://github.com/nmap/nmap/blob/master/libpcap/CMakeLists.txt

**Pull request merged**